docs/conf.py

```diff
@@ -227,6 +228,8 @@ def setup(app):
    :alt: Print
 .. |export| image:: /img/export.png
    :alt: Export
+.. |exams| image:: /img/quiz-icon.png
+   :alt: Quizzes
```

Comment:
see https://github.com/learningequality/kolibri-docs/pull/202 as an alternative to manually creating these